src/main/groovy/com/bmuschko/gradle/docker/DockerExtension.groovy

```diff
@@ -33,6 +33,7 @@ class DockerExtension {
     public DockerExtension(Project project) {
         this.project = project
         this.url = getDefaultDockerUrl()
+        this.certPath = this.url.startsWith('tcp://') ? new File(System.getenv("DOCKER_CERT_PATH")) : null
```

Comment:
If the user has this env-var set, for whatever reason, lets just set it here and do something like:
```
def certPathEnvVar = System.getenv("DOCKER_CERT_PATH")
if (certPathEnvVar) {
    this.certPath = project.file(certPathEnvVar)
    if (!this.certPath.exists()) {
        throw new GradleException("Could not find DOCKER_CERT_PATH @ ${certPathEnvVar}")
    }
}
```

Reply:
@cdancy do we nececerily need an exception in the constructor?
please see my followup.

missing disclamer: I'm not a groovy dev at all.